src/js/components/stock-movement-wizard/outbound/EditPage.jsx

```diff
@@ -246,8 +246,7 @@ const FIELDS = {
         getDynamicAttr: ({
           fieldValue, subfield, reasonCodes, updateRow, values, rowIndex,
         }) => {
-          const isSubstituted = values.lineItems && values.lineItems[rowIndex] &&
-            values.lineItems[rowIndex].statusCode === 'SUBSTITUTED';
+          const isSubstituted = fieldValue && fieldValue.statusCode === 'SUBSTITUTED';
```

Comment:
@kchelstowski  Please take a look at `request/EditPage` if it is proper there too

Reply:
@awalkowiak nice catch, I've forgotten about requests - obviously there was the same issue - below I commited a fix for requests also